python3/httplib2/socks.py

```diff
@@ -207,7 +207,7 @@ def __rewriteproxy(self, header):
 
     def __getauthheader(self):
         auth = self.__proxy[4] + ":" + self.__proxy[5]
-        return "Proxy-Authorization: Basic " + base64.b64encode(auth)
+        return "Proxy-Authorization: Basic " + base64.b64encode(auth.encode('utf-8')).decode()
```

Comment:
It would seem reasonable to encode/decode using same encoding. `auth.encode()).decode()`

Reply:
Yes that makes sense.